liquibase-core/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -631,6 +631,7 @@ protected Object readDefaultValue(CachedRow columnMetadataResultSet, Column colu
             Object columnDef = columnMetadataResultSet.get(COLUMN_DEF_COL);
             if ("CHAR".equalsIgnoreCase(columnInfo.getType().getTypeName()) && (columnDef instanceof String) && !
```

Comment:
I think this could benefit from a comment explaining why we are doing this. Something like "If the column type is a CHAR and the column definition is a non-quoted string, then we need to set the COL_DEF_COL to null so that the generated column definition doesn't get treated as a computed value." (I think that is a correct short summary of the change required, but fix the comment if needed). Remember that anyone from the open-source world will not have access to the Datical JIRA ticket, so we can't depend on that for additional info.

Reply:
done